book/website/project-design/sdp/pd-sdp-private-learning.md

```diff
@@ -0,0 +1,51 @@
+(pd-sdp-private-learning)=
+
+# Privacy-preserving Machine learning
+
+Much current research in data science involves machine learning (ML) models interacting with data sourced from a large number of individuals with significant variation in the general level of awareness, consent, and understanding of research goals. As such, researchers have a responsibility to protect the confidentiality and privacy of the people whose data is being processed. At the same time, sharing both data and trained models drives scientific advancement and promotes important social goals in open and transparent science.
+
+It's important to note that local and international regulations such as the [General Data Protection Regulation](https://gdpr-info.eu/) (GDPR) and the EU’s policy on [trustworthy AI](https://digital-strategy.ec.europa.eu/en/policies/european-approach-artificial-intelligence) also establish legal duties and principles on privacy protection that the following tools may help researchers meet.
+
+## Sharing data with privacy
+
+Training a complex ML model can often require a very large amount of data, more than a single researcher or organisation could feasibly generate. Sharing our data not only helps us to create more reproducible research, but promotes advancements in the field as a whole. However, this does pose the risk of inadvertently sharing personal information that could be used to identify a subject.
+
+Most researchers will remove uniquely identifying information (such as ID numbers, address, and phone numbers) before publication, but recent research has shown that with access to secondary datasets, such 'pseudonymised' datasets can still be traced back to the individual {cite:ps}`narayananRobustDeanonymizationLarge2008,sunIdentityAnonymizationHighdimensional2012`.
+
+### Differential privacy
+
+Differential privacy is a statistical tool which can estimate the risk of uniquely identifying a member of a dataset, whereupon calibrated noise can be added to ensure that privacy is preserved {cite:ps}`yangDifferentialPrivacyData2012`.
+
+### Synthetic data generation
+
+If sharing the original data raises privacy or ethical concerns, we can still contribute useful information by sharing synthetic datasets that reproduce statistical features of the original dataset without exposing actual instances {cite:ps}`torfiDifferentiallyPrivateSynthetic2020`. 
```

Comment:
I might know two case studies here: one from neuroimaging/sharing brain data, one from health economics. We were thinking about synthetic data for  a neuroimaging study, at the moment synthetic data creates a lot of noise and bias to the data - maybe that could be a caveat

Reply:
Please, feel free to add any relevant case studies that you're aware of. The descriptions of each of the mechanisms also requires significant expansion, ideally with an overview of current approaches - so for synthetic data for instance, there might be an aside about GANs and something about Gaussian or Bayesian methods. Actually, this [Data Study Group report](https://www.turing.ac.uk/research/publications/data-study-group-final-report-department-work-and-pensions) I contributed to might be worth including.